src/components/blm/people/index.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+
+// semantic-ui
+import {
+  Card,
+  Container,
+  Divider,
+  Header,
+  Image,
+  Button,
+} from 'semantic-ui-react';
+
+// components
+import PeopleCards from './../../common/PeopleCards';
+
+// card content
+import {
+  activists,
+  stories,
+} from './cards';
+
+
+class UyghurPeople extends Component {
+  render() {
+    return (
+      <div>
+        <Header as="h1">
+          People of the Uyghur Movement
+        </Header>
+        <Divider />
+
+        
+
```

Comment:
remove all this unnecessary space, just have one space at most for ease of reading